### PR TITLE
Add more Romi docs

### PR DESCRIPTION
**Pull request by zhiquanyeo**

### CODE REVIEW

File: source/docs/romi-robot/web-ui.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+The Romi WPILibPi image ships with the latest (at publication time) version of the Romi web service. To support upgrading to newer versions of the Romi web service, this section allows users to upload a pre-built bundle that can be obtained via the Romi web service `GitHub releases page <https://github.com/wpilibsuite/wpilib-ws-robot-romi/releases>`__.
+
+To perform an upgrade, download the appropriate .tgz file from the GitHub Releases page. Next, select the downloaded .tgz file and click Save. The updated web service bundle will be uploaded to the Raspberry Pi, and be installed. After a short moment, the Romi Status section should update itself with the latest version information.
+
+External IO Configuration
+-------------------------
+
+.. image:: images/web-ui/romi-ui-io-config.png
+   :alt: External IO Configuration
+
```
Comment (Daltz333) [inline source/docs/romi-robot/web-ui.rst:50]: Extra line  here

File: source/docs/romi-robot/web-ui.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+The "Robot Port" row provides the appropriate WPILib mapping for each configured GPIO channel. For example, EXT 0 is configured as a Digital IO channel, and will be accessible in WPILib as a DigitalInput (or DigitalOutput) channel 8.
+
+IMU Calibration
+---------------
+
+.. image:: images/web-ui/romi-ui-imu-calibration.png
+   :alt: IMU Calibration
+
+.. note:: The Raspberry Pi must be in Writable mode for this section to work.
```
Comment (Daltz333) [inline source/docs/romi-robot/web-ui.rst:66]:
```suggestion
.. note:: The Raspberry Pi must be in **Writable** mode for this section to work.
```

File: source/docs/romi-robot/web-ui.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+This section allows users to calibrate the gyro on the Romi. Gyros usually have some sort of zero-offset error, and calibration allows the Romi to calculate the offset and use it in calculations.
+
+To begin calibration, place the Romi on a flat, stable surface. Then, click the "Calibrate Gyro" button. A progress bar will appear, showing the current calibration process. Once calibration is complete, the latest offset values will be displayed on screen and registered with the Romi web service.
```
Comment (Daltz333) [inline source/docs/romi-robot/web-ui.rst:70]:
```suggestion
To begin calibration, place the Romi on a flat, stable surface. Then, click the :guilabel:`Calibrate Gyro` button. A progress bar will appear, showing the current calibration process. Once calibration is complete, the latest offset values will be displayed on screen and registered with the Romi web service.
```

File: source/docs/romi-robot/getting-to-know-romi.rst [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- 2x geared motors with encoders
+- 1x Inertial Measurement Unit (IMU)
+- 3x LEDs (green, yellow, red)
+- 3x pushbuttons (marked A, B and C)
+- 5x configurable GPIO channels
+
+Motors, Wheels and Encoders
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The motors used on the Romi have a 120:1 gear reduction, and a no-load speed of 150 RPM at 4.5V. The wheels have a diameter of 70mm (2.75").
```
Comment (AustinShalit) [inline source/docs/romi-robot/getting-to-know-romi.rst:20]: We should include all 4 DC motor characteristics (free speed, stall torque, stall current, and free current):

```suggestion
The motors used on the Romi have a 120:1 gear reduction, and a no-load output speed of 150 RPM at 4.5V. The free current is 0.13 amps and the stall current is 1.25 amps. Stall torque is 25 oz-in (0.1765 N-m) but the built-in safety clutch might start slipping at lower torques. The wheels have a diameter of 70mm (2.75").
```

**Review by AustinShalit — CHANGES_REQUESTED** [review]

File: source/docs/romi-robot/getting-to-know-romi.rst [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+- 2x geared motors with encoders
+- 1x Inertial Measurement Unit (IMU)
+- 3x LEDs (green, yellow, red)
+- 3x pushbuttons (marked A, B, and C)
+- 5x configurable GPIO channels
+
+Motors, Wheels and Encoders
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The motors used on the Romi have a 120:1 gear reduction, and a no-load output speed of 150 RPM at 4.5V. The free current is 0.13 amps and the stall current is 1.25 amps. Stall torque is 25 oz-in (0.1765 N-m) but the built-in safety clutch might start slipping at lower torques. The wheels have a diameter of 70mm (2.75").
```
Comment (AustinShalit) [inline source/docs/romi-robot/getting-to-know-romi.rst:20]:
```suggestion
The motors used on the Romi have a 120:1 gear reduction, and a no-load output speed of 150 RPM at 4.5V. The free current is 0.13 amps and the stall current is 1.25 amps. Stall torque is 25 oz-in (0.1765 N-m) but the built-in safety clutch might start slipping at lower torques.

The wheels have a diameter of 70mm (2.75"). They have a measured trackwidth of 145mm (5.71").
```

**Review by AustinShalit — CHANGES_REQUESTED** [review]

**Review by AustinShalit — APPROVED** [review]

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**